woocommerce/woocommerce-ios
# [Mobile Payments] CoD onboarding design changes

### Description

In discussions we have agreed some slight changes to the design of the Cash on Delivery prompt screen:

1. Use the `woo-customer-waiting` image for the error
2. Provide specific Learn More text with a link directly to the section of the docs which explains the use of Cash on Delivery.

Additionally, this PR addresses the non-blocking issues raised in during the review of #7499

### Testing instructions


Using a store with CoD disabled (check that it's disabled in `WP Admin > WooCommerce > Settings > Payments`)

1. Go to the hub menu
2. Tap Payments and wait for the spinner to complete
3. The onboarding incomplete footer message should appear: tap `Continue Setup`
4. The `Add Pay in Person to your checkout` onboarding screen should show


Changes:
1. Observe that the image has changed to a person leaning on a pile of boxes.
2. Check that the Learn More link opens the docs at the "Next, add a Cash on Delivery Payment Method" section.

### Screenshots



https://user-images.githubusercontent.com/2472348/185375378-9ba82345-2f97-4abc-bb3a-1d9ce1e2fe78.mp4

#### Before
![before tweaks](https://user-images.githubusercontent.com/2472348/185376489-9faf70a1-8d2a-4e55-b959-a9127ac069ed.jpg)





---
